src/pages/tezos/cryptobot/[id].js

```diff
@@ -248,7 +248,11 @@ function BotView({ location }) {
             >
               {claimButtonDisabled ? (
                 <div className={`flex justify-center align-center`}>
-                  <Loader type="ThreeDots" color="#BFDBFE" height={28} />
+                  {getUserBalance.value > 0.5 ? (
+                    <Loader type="ThreeDots" color="#BFDBFE" height={28} />
+                  ) : (
+                    'Low Balance'
```

Comment:
Here I think rather than using `Low Balance` text, using `claim` makes sense since we are not showing an info here but it's an actual button with an action attached to it.

Reply:
Yes, I agree. But labelling the button with the reason of why it's disabled felt more intuitive to me. 
I could be wrong here tho.

What do you think is optimal @harsh242 - Button text should be "Confirm" or "Low Balance" in case the button's disabled because the user doesn't have sufficient XTZ?

Reply:
@harsh242  ☝️

Reply:
Keep the button text as "Confirm only", for low balance we were showing them the error right?

Reply:
okay